File: Front/facturas-app/types/factura.ts
export interface Factura {
  facturaID: number;
  fecha: string;
  monto: number;
  estado: string;
  descripcion?: string;
}

export interface FacturasResponse {
  clienteID: number;
  facturas: Factura[];
  totalFacturas: number;
  montoTotal: number;
}

File: Back/FacturasAPI/FacturasAPI/Models/Factura.cs
namespace FacturasAPI.Models;

public class Factura
{
    public int FacturaID { get; set; }
    public int ClienteID { get; set; }
    public DateTime Fecha { get; set; }
    public decimal Monto { get; set; }
    public string Estado { get; set; } = string.Empty;
    public string? Descripcion { get; set; }
}
